nltk/test/conftest.py

```diff
@@ -15,12 +15,15 @@ def mock_plot(mocker):
         pass
 
 
-@pytest.fixture(scope="session", autouse=True)
-def teardown_loaded_corpora(request):
+@pytest.fixture(scope="module", autouse=True)
+def teardown_loaded_corpora():
     """
     After each test session ends (either doctest or unit test),
     unload any loaded corpora
     """
+
+    yield  # first, wait for the test to end
```

Comment:
this was my bad (oops) this fixture was running before the tests rather than after. ([Relevant pytest docs.](https://docs.pytest.org/en/stable/fixture.html#yield-fixtures-recommended)) Also I changed it from "session" to "module" because waiting till the end of all the tests to teardown is slower than doing smaller teardowns after each test. I'm not totally sure why.